app/services/seek_and_destroy_expired_dossiers_service.rb

```diff
@@ -0,0 +1,11 @@
+class SeekAndDestroyExpiredDossiersService
```

Comment:
Pour faire moins Terminator™,

```suggestion
class ExpiredDossiersDeletionService
```

?